src/util/actor.js

```diff
@@ -107,15 +109,21 @@ class Actor {
                 cancel();
             }
         } else {
-            // Store the tasks that we need to process before actually processing them. This
+            // In workers, store the tasks that we need to process before actually processing them. This
             // is necessary because we want to keep receiving messages, and in particular,
             // <cancel> messages. Some tasks may take a while in the worker thread, so before
             // executing the next task in our queue, postMessage preempts this and <cancel>
             // messages can be processed. We're using a MessageChannel object to get throttle the
             // process() flow to one at a time.
             this.tasks[id] = data;
             this.taskQueue.push(id);
-            this.invoker.trigger();
+            if (this.isWorker) {
```

Comment:
Thanks for your contribution @msbarry ! :)
Could you please change this to use the `isWorker()` function in `src/util.js`. Code looks good otherwise.

I'll post browser testing reports here.

Reply:
Thanks! Just switched to `isWorker()`